### PR TITLE
add comments

### PR DESCRIPTION
**Pull request by ritamma**

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+class CommentsController < ApplicationController
+
+ def create
+    @lesson = Lesson.find(params[:lesson_id])
```
Comment (fooooxmr) [inline app/controllers/comments_controller.rb:4]: You can move to the callback.

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+     redirect_to lesson_path(@lesson)
+   else
+     render :edit, status: :unprocessable_entity
+   end
+ end
+
+  def destroy
+    @lesson = Lesson.find(params[:lesson_id])
+    @comment = @lesson.comment.find(params[:id])
+    return render :file => "public/403.html", :status => :unauthorized if @comment.user != current.user
```
Comment (fooooxmr) [inline app/controllers/comments_controller.rb:29]: It's ok for now, but in future, we will have admin, who should be able to destroy any comments.

File: app/controllers/comments_controller.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
+class CommentsController < ApplicationController
+
+ def create
+    @lesson = Lesson.find(params[:lesson_id])
+    @comment = @lesson.comments.create!(comment_params.merge(user:current_user))
```
Comment (fooooxmr) [inline app/controllers/comments_controller.rb:5]: Maybe better write `current_user.comments.create(comment_params.merge(lesson: @lesson)`?

File: db/migrate/20220827155600_create_comments.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :commenter
```
Comment (fooooxmr) [inline db/migrate/20220827155600_create_comments.rb:4]: Why do we need commenter field, if commenter it's a user, who added comment? I think, we can remove this field.

**Review by fooooxmr — CHANGES_REQUESTED** [review]

File: app/controllers/comments_controller.rb (hunk 2 above)
Comment (ritamma) [reply app/controllers/comments_controller.rb:29]: Should I comment out or delete this line for now?

File: public/401.html [hunk 5]
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Error(401)</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <style>
```
Comment (Terapool) [inline public/401.html:6]: Lets leave this styles as is, but later we will move them ito separate file

**Review by Terapool — APPROVED** [review]

File: public/401.html (hunk 5 above)
Comment (ritamma) [reply public/401.html:6]: okay

File: app/controllers/comments_controller.rb (hunk 2 above)
Comment (fooooxmr) [reply app/controllers/comments_controller.rb:29]: We can leave it for now.

File: db/migrate/20220827155600_create_comments.rb (hunk 4 above)
Comment (fooooxmr) [reply db/migrate/20220827155600_create_comments.rb:4]: Any updates here?

Comment (ritamma) [reply db/migrate/20220827155600_create_comments.rb:4]: I made a migration to remove the column. Should I remove the line directly in the migration?

Comment (fooooxmr) [reply db/migrate/20220827155600_create_comments.rb:4]: Oh, it's fine. I have. not seen it.

**Review by fooooxmr — APPROVED** [review]

**Review by evrusha — APPROVED** [review]

**Review by Dmitryshl — APPROVED** [review]

**Pull request merged**